Added a boolean in msal-node's system options, which disables built-in retry policies (#6920)
AzureAD/microsoft-authentication-library-for-js: lib/msal-node/docs/configuration.md

| `proxyUrl`               | The URL of the proxy the app is running behind                                                                                                                                                                                                       | string                                                                                                                     | Empty string `""`                                                                                                                    |
| `customAgentOptions`     | Set of configurable options to set on a http(s) agent                                                                                                                                                                                                | Object - [NodeJS documentation on alloweable options](https://nodejs.org/docs/latest-v16.x/api/http.html#new-agentoptions) | Empty Object `{}`                                                                                                                    |

> | `disableInternalRetries` | A boolean that, if enabled, disables MSALJS's built-in retry policies, allowing the app developer to specify their own retry policy. Currently, only Managed Identity flows have a simple retry policy of retrying once on certain HTTP error codes. | boolean                                                                                                                    | boolean `false`                                                                                                                      |

Author: only the "disableInternalRetries" line was changed here

> | `disableInternalRetries` | A boolean that, if enabled, disables MSALJS's built-in retry policies, allowing the app developer to specify their own retry policy. Currently, only Managed Identity flows have a simple retry policy of retrying once on certain HTTP error codes. | boolean                                                                                                                    | boolean `false`                                                                                                                      |

Reviewer: ```suggestion
| `disableInternalRetries` | A flag that disables MSALJS's built-in retry policies, allowing the app developer to specify their own retry policy. Currently, only Managed Identity flows have a retry policy. | boolean                                                                                                                    | boolean `false`                                                                                                                      |
```
